test/e2e/pkg/testnet.go

```diff
@@ -826,3 +821,12 @@ func parseCsv(csvString string) ([][]string, error) {
 
 	return records, nil
 }
+
+func ParseKeyValueField(name string, field string) (key string, value string, err error) {
+	tokens := strings.Split(field, " = ")
+	if len(tokens) != 2 {
+		return key, value, fmt.Errorf("invalid '%s' field: \"%s\", "+
+			"expected \"key = value\"", name, field)
+	}
+	return tokens[0], tokens[1], nil
```

Comment:
```suggestion
	tokens := strings.Split(field, "=")
	if len(tokens) != 2 {
		return key, value, fmt.Errorf("invalid '%s' field: \"%s\", "+
			"expected \"key = value\"", name, field)
	}
	return strings.TrimSpace(tokens[0]), strings.TrimSpace(tokens[1]), nil
```

We should allow the `key=value` entries (without enforcing space between the '='), these would be valid and written as `key = value` in the `config.toml` file

Should also change in line 539
```
	for _, entry := range n.Config {
		_, _, err := ParseKeyValueField("config", entry)
		if err != nil {
			return err
		}
	}
```